composer.json

```diff
@@ -14,7 +14,7 @@
         "symfony/options-resolver": "^3.2",
         "symfony/yaml": "^3.2",
         "doctrine/dbal": "2.5.10",
-        "doctrine/migrations": "1.0.0-alpha3",
+        "doctrine/migrations": "^1.0.0",
```

Comment:
i think we want to put "doctrine/migrations": "^1.5.0" here, since the minimum version we want installed is 1.5